.azure-pipelines/client.test.live.yml

```diff
@@ -10,7 +10,7 @@ queue:
 steps:
 - bash: |
    LOG_PARAMS='-Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
-   mvn test -Dhttp.keepAlive=false -Dsurefire.rerunFailingTestsCount=3 $LOG_PARAMS -f $(pomFile)
+   mvn test -Dmaven.wagon.http.pool=false -Dsurefire.rerunFailingTestsCount=3 $LOG_PARAMS -f $(pomFile)
```

Comment:
Please add a comment explaining why `maven.wagon.http.pool` is being set to `false`.

Reply:
Added the comment.